lib/lower/PFTBuilder.cpp

```diff
@@ -301,8 +312,8 @@ void annotateEvalListCFG(PFT::EvaluationCollection &evaluationCollection,
   for (auto &eval : evaluationCollection) {
     eval.isTarget = nextIsTarget;
     nextIsTarget = false;
-    if (eval.isConstruct()) {
-      annotateEvalListCFG(*eval.getConstructEvals(), &eval);
+    if (auto *subs{eval.getConstructEvals()}) {
+      annotateEvalListCFG(*subs, &eval);
```

Comment:
Was this the bug? Don't all constructs have a body (with 0 or more evals)?

Reply:
Well, there was one bug here that I did not mentioned in the commit, not sure if it came from the refactoring, the issue was that `eval.isConstruct()` and `eval.isStmt()` were not equivalent so `eval.isConstruct()` was not ensuring `eval.getConstructEvals()` was not null and that led to weird things, like visiting `ConstructStmt` as if they were constructs. The `eval.isConstruct()` is fixed, but it feels safer to test the pointer here as a user.

Reply:
OK. Thanks for fixing the bug!